dmitryi_bondar/task_3_OS_Linux/killercoda_linux/backup_project_script.sh

```
@@ -0,0 +1,65 @@
+: << 'COMMENT'
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)